applications/dashboard/settings/class.hooks.php

```diff
@@ -263,7 +263,7 @@ public function SiteNavModule_all_handler($sender) {
       if (Gdn::Session()->IsValid()) {
          // Switch between the full site and mobile.
          if (IsMobile()) {
-            $sender->addLink('etc.nomobile', array('text' => t('Full Site'), 'url' => '/profile/nomobile', 'icon' => icon('resize-full'), 'sort' => 100));
+            $sender->addLink('etc.nomobile', array('text' => t('Full Site'), 'url' => Url('/profile/nomobile'), 'icon' => icon('resize-full'), 'sort' => 100));
```

Comment:
I don't even see the bug on the site that there is supposed to be a bug. You should write about the bug you are fixing in the description of the PR.